### PR TITLE
implement cancel command properly

### PR DESCRIPTION
**Pull request by rntz**

### CODE REVIEW

File: code/cancel.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from talon import speech_system, actions
+import logging
+
+cancel_phrase = "cancel cancel".split()
+
+def pre_phrase(d):
+    logging.info(d)
```
Comment (rntz) [inline code/cancel.py:7]: Urk, I should remove this line.

File: misc/cancel.talon [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# the actual behavior of "cancel cancel" is implemented in code/cancel.py
+# it allows you to prevent a command from executing by ending it with "cancel cancel"
```
Comment (pokey) [inline misc/cancel.talon:2]: Maybe a quick NB that if they want to use a different phrase they also need to change it in the Python file? Alternatively could unify to single source of truth with a capture

**Review by pokey — APPROVED** [review]
LGTM

File: misc/cancel.talon (hunk 2 above)
Comment (rntz) [reply misc/cancel.talon]: Added a note

**Pull request merged**